### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jordan-sullivan** [jordan-sullivan/travel-tracker#17]
Create a branch for accessibility.
Use this branch to make your dashboard as accessible as possible.
Push this branch up to GH. You can merge the changes into main but do not delete this branch.
This branch should not have a login page so that during the live eval, we can run the Lighthouse audit and check tabbability of your dashboard (without the login page).

**Comment by jordan-sullivan** [comment]
UI/UX & Accessibility

4: Application has clearly had special consideration around accessibility and usability on devices. Lighthouse accessibility audit is at a 100% and application is fully tabbable.
3: Application has many strong pages/interactions. The application can stand on its own to be used by instructor without guidance. UI does not detract from the UX. Lighthouse accessibility audit is at least 90% and application is fully tabbable.
2: The application may be confusing or difficult to use at times. The UI is incomplete, Accessibility has been considered, but does not have strong accessible features.
1: Application is confusing or difficult to use. The UI is incomplete. Accessibility has not been considered.
